spec/models/user_spec.rb

```diff
@@ -17,6 +17,29 @@
     end.to change(ActionMailer::Base.deliveries, :count).by(1)
   end
 
+  describe 'password validations' do
+    it 'disallows common words' do
+      user = create(:user)
+      user.password = 'password'
+
+      expect(user.valid?).to eq false
+      expect(user.errors[:password].first).to match('not strong enough')
```

Comment:
I don't think this is worth testing. I think it would be worth testing against the entropy level configured instead.